change_ttl/README.md

```diff
@@ -1,3 +1,56 @@
-# Tinybird Versions - {{ YOUR USE CASE NAME HERE }}
+# Tinybird Versions - Change a Data Source's TTL
+
+
+The Data Source's TTL determines how long data remains valid and accessible within a system. This process is crucial as it ensures data accuracy, relevance, and helps in managing system performance by refreshing or removing outdated information, thereby maintaining the integrity and efficiency of your data infrastructure.
+
+These are the steps in modifying TTL, ensuring there's no data loss during the process.
+
+## Step 1: Change the ENGINE_TTL setting in a Data Source
```

Comment:
IMO, If there is only one step, we shouldn't say `Step 1`, WDYT @elenatorro ?

Reply:
Agree! I'll add it to my WIP changes 👍🏽